### PR TITLE
Add documentation custom field item for name retrieval

### PR DESCRIPTION
**Pull request by kkahla**
#what Include documentation on retrieving custom field items by name including a name with a wildcard. Added example of this as well

#why New functionality and it needs to be documented

### CODE REVIEW

File: source/includes/APIReference/CustomFieldItems/Examples/_retrieve.tmpl.erb [hunk 1]
```diff
@@ -32,3 +32,12 @@
     :response_body => @response_body
```
Comment (dustinsweet) [inline source/includes/APIReference/CustomFieldItems/Examples/_retrieve.tmpl.erb]: Remove `response_body` param.  Only include in the last example (your new one).

**Review by dustinsweet — CHANGES_REQUESTED** [review]
Just one minor fix needed.  Please also add an entry to _changelog.md.erb.  Thanks Katy!

File: source/includes/APIReference/CustomFieldItems/Examples/_retrieve.tmpl.erb (hunk 1 above)
Comment (kkahla) [reply source/includes/APIReference/CustomFieldItems/Examples/_retrieve.tmpl.erb]: Done

**Comment by kkahla** [comment]
@dustinsweet This is ready for review again :)

**Review by dustinsweet — APPROVED** [review]

**Pull request merged**